app/Services/CalculateJobTimeService.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use Carbon\Carbon;
+
+class CalculateJobTimeService
+{
+    public function calculate(string $startedAt, string $completedAt): int
+    {
+        $startedAt = Carbon::create($startedAt); 
+        $completedAt = Carbon::create($completedAt);
+
+        $diffInSeconds = $startedAt->diffInSeconds($completedAt);
+
+        $diffInMinutes = (int)ceil($diffInSeconds / 60);
```

Comment:
Carbon should have method to diff in minutes with rounding value, I think so.

Reply:
`diffInMinutes` rounds value, but here the value has to be always rounded **up**